projects/common/src/index.ts

```diff
@@ -13,15 +13,15 @@ export const articlePillars = [
     'lifestyle',
     'neutral',
 ] as const
-export const articleTypes = [
-    'article',
-    'review',
-    'opinion',
-    'longread',
-] as const
+
+export enum ArticleType {
+    Longread = 'longread',
+    Article = 'article',
+    Review = 'review',
+    Opinion = 'opinion',
+}
```

Comment:
Not sure why this is an enum

Reply:
Hi! I explained it at the bottom of the PR text, sorry, it's hard to see with the image in the middle :)

> Turned the article types into an enum, the idea is that we can send these from the backend like we do with front cards

Reply:
But this could be sent from the backend as a string union type as well though, so why a union?

Reply:
it's just a bit cleaner than passing strings around? for the fronts what i did was that the enum values were more abstract and uppercase, happy to do that

Reply:
The enum would give you an object you can use for lookups, but they'll both get serialised into strings. The `[`...`]` as const` also gives you an array you can call `find` on to take a string to the union type safely.

Reply:
Yes soz I wasn't making myself clear. I just mean it's neater to abstract the actual value the backend sends as a string into this enum with key names than to use the literal strings as a union type

Reply:
As they have the same meanings, all the enums end up doing here are causing you to need `getEnumPosition` where with string unions you wouldn't need to.

Reply:
`getEnumPosition` is just a quirky workaround for the devtools. Since this is just for our eyes only I'm not overly worried (it's bad practice anyway, objects shouldn't be accessed by index). If `getEnumPosition` is the problem I'm happy to rewrite that part a bit more cleanly?